com.opentabletdriver.OpenTabletDriver.yaml

```diff
@@ -0,0 +1,69 @@
+app-id: com.opentabletdriver.OpenTabletDriver
```

Comment:
the domain `opentabletdriver.com` doesn't exist. The app-id doesn't meet the requirements.

Reply:
https://opentabletdriver.net/